projects/detector/private/pybindings/detector.cxx

```diff
@@ -19,11 +21,78 @@ PYBIND11_MODULE(detector,m) {
     .def(init<std::string const &, std::string const &, std::string const &>())
     .def("LoadEarthModel",&EarthModel::LoadEarthModel)
     .def("LoadMaterialModel",&EarthModel::LoadMaterialModel)
-    .def("GetSectors",&EarthModel::GetSectors);
+    .def("GetSectors",&EarthModel::GetSectors)
+    // .def("GetColumnDepthInCGS",overload_cast<LI::geometry::Geometry::IntersectionList const &, LI::math::Vector3D const &, LI::math::Vector3D const &>(&EarthModel::GetColumnDepthInCGS))
+    // .def("GetColumnDepthInCGS",overload_cast<LI::math::Vector3D const &, LI::math::Vector3D const &>(&EarthModel::GetColumnDepthInCGS))
+    // .def("GetInteractionDepthInCGS",overload_cast<LI::geometry::Geometry::IntersectionList const &, LI::math::Vector3D const &, LI::math::Vector3D const &, std::vector< LI::dataclasses::Particle::ParticleType > const &, std::vector< double > const &, double const &>(&EarthModel::GetInteractionDepthInCGS))
+    // .def("GetInteractionDepthInCGS",overload_cast<LI::math::Vector3D const &, LI::math::Vector3D const &, std::vector< LI::dataclasses::Particle::ParticleType > const &, std::vector< double > const &, double const &>(&EarthModel::GetInteractionDepthInCGS))
+    // .def("DistanceForColumnDepthFromPoint",&EarthModel::DistanceForColumnDepthFromPoint)
+    // .def("DistanceForColumnDepthToPoint",&EarthModel::DistanceForColumnDepthToPoint)
+    // .def("DistanceForInteractionDepthFromPoint",&EarthModel::DistanceForInteractionDepthFromPoint)
+    // .def("DistanceForInteractionDepthToPoint",&EarthModel::DistanceForInteractionDepthToPoint) 
+    // .def("DistanceForColumnDepthFromPoint",overload_cast<LI::geometry::Geometry::IntersectionList const &, LI::math::Vector3D const &, LI::math::Vector3D const &, double>(&EarthModel::DistanceForColumnDepthFromPoint))
+    // .def("DistanceForColumnDepthFromPoint",overload_cast<LI::math::Vector3D const &, LI::math::Vector3D const &, double>(&EarthModel::DistanceForColumnDepthFromPoint))
+      
+    // .def("DistanceForColumnDepthFromPoint", static_cast<double (EarthModel::*)(LI::geometry::Geometry::IntersectionList const &, LI::math::Vector3D const &, LI::math::Vector3D const &, double)>(&EarthModel::DistanceForColumnDepthFromPoint))
+    // .def("DistanceForColumnDepthFromPoint", static_cast<double (EarthModel::*)(LI::math::Vector3D const &, LI::math::Vector3D const &, double)>(&EarthModel::DistanceForColumnDepthFromPoint))
```

Comment:
Are these waiting to be implemented, or is there another reason they are commented out?

Reply:
I think I once wanted to implement them but I don't think they're necessary, we should just delete them